.markdownlint-cli2.jsonc

```diff
@@ -0,0 +1,9 @@
+{
+  "config": {
+    "default": true,
+    "MD013": false, // Ignore line-length
+    "MD045": false, // Ignore alt text requirement for images
```

Comment:
This should probably be a TODO to fix later, but the current ones didn't have a text either

Reply:
https://github.com/github/github-ospo/issues/18